pkg/pillar/hypervisor/containerd.go

```diff
@@ -206,7 +208,7 @@ func (ctx ctrdContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
 			} else {
 				usedMemPerc = 0
 			}
-			cpuTotal = metric.CPU.Usage.Total / 1000000000
+			cpuTotal = metric.CPU.Usage.Total / 1000000000 / clockTicks
```

Comment:
Would be more readable if you introduced a 
pkg/pillar/cmd/zedrouter/appcontainer.go:const nanoSecToSec uint64 = 1000000000